src/commands/title-escrow/accept-surrendered.ts

```diff
@@ -0,0 +1,47 @@
+import { Argv } from "yargs";
+import { error, success, info } from "signale";
+import { getLogger } from "../../logger";
+import { withGasPriceOption, withNetworkAndKeyOption } from "../shared";
+import { TitleEscrowSurrenderDocumentCommand } from "./title-escrow-command.type";
+import { acceptSurrendered } from "../../implementations/title-escrow/accept-surrendered";
+import { getEtherscanAddress } from "../../utils";
+
+const { trace } = getLogger("surrender:title-escrow");
+
+export const command = "accept-surrendered [options]";
+
+export const describe = "Accepts a surrendered transferable record on the blockchain";
+
+export const builder = (yargs: Argv): Argv =>
+  withGasPriceOption(
+    withNetworkAndKeyOption(
+      yargs
+        .option("token-registry", {
+          alias: "r",
+          description: "Address of the token registry that the transferable record was issued from",
+          type: "string",
+          normalize: true,
```

Comment:
do we need `normalize`?

Reply:
dont need, since this input wont be a path. ive removed it.